vhdbuilder/packer/install-dependencies.sh

```diff
@@ -616,9 +616,9 @@ if [ "$OS" = "$UBUNTU_OS_NAME" ]; then
   apt_get_purge 20 30 120 snapd || exit 1
   apt_get_purge 20 30 120 apache2-utils || exit 1
 
-  apt-get -y autoclean || exit 1
-  apt-get -y autoremove --purge || exit 1
-  apt-get -y clean || exit 1
+  apt-get -o DPkg::Lock::Timeout=60 -y autoclean || exit 1
```

Comment:
same here can't we drop the apt.conf file on the VM were all those scripts are executed ?

Reply:
Removed the new apt arguments.